### PR TITLE
Update readme in DS2

### PR DESCRIPTION
**Pull request by kuke**
Remove the limitation of cuDNN version due to [the bug fix](https://github.com/PaddlePaddle/Paddle/pull/3309)

### CODE REVIEW

File: deep_speech_2/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ### Prerequisites
 
  - **Python = 2.7** only supported;
- - **cuDNN >= 6.0** is required to utilize NVIDIA GPU platform in the installation of PaddlePaddle, and the **CUDA toolkit** with proper version suitable for cuDNN. The cuDNN library below 6.0 is found to yield a fatal error in batch normalization when handling utterances with long duration in inference.
+ - **cuDNN** library is required to utilize NVIDIA GPU platform sufficiently in the installation of PaddlePaddle, and the **CUDA toolkit** with proper version suitable for cuDNN.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: If Python 2.7 and cuDNN is already required when installing PaddlePaddle, we don't have to claim them again.

**Review by xinghai-sun — APPROVED** [review]
I recommend to remove the whole `Prerequisites`.

File: deep_speech_2/README.md (hunk 1 above)
Comment (pkuyym) [inline deep_speech_2/README.md]: maybe we can specify some version for ```proper version```

Comment (kuke) [reply deep_speech_2/README.md]: Agree with that

**Pull request merged**